README.md

```diff
@@ -62,3 +62,14 @@ at the root of your project
 ## Routes
 
 Check the application routes with `mix pheonix.routes`
+
+## Further Documentation
+
+You can view further technical documentation by running:
+```bash
```

Comment:
Thanks @iteles, I think @jackcarlisle and I originally saw this on [this udemy tutorial](https://www.udemy.com/the-complete-elixir-and-phoenix-bootcamp-and-tutorial/) and I found [this](https://brainlid.org/elixir/2016/09/01/phoenix-project-documentation.html) really useful for an introduction on how to document in a phoenix project.

I'm going to play with it over the next few days in this project, then maybe write up my findings in [dwyl/learn-phoenix-framework](https://github.com/dwyl/learn-phoenix-framework)